src/main/java/frc/robot/parameters/ElevatorParameters.java

```diff
@@ -8,31 +8,50 @@
 package frc.robot.parameters;
 
 import frc.robot.subsystems.Elevator;
+import frc.robot.util.MotorDirection;
 
 public enum ElevatorParameters {
-  PracticeBase2025(3.65, Elevator.PRACTICE_BOT_GEAR_RATIO, 0, 1.39, 14, false),
-  CompetitionBase2025(3.65, Elevator.COMPETITION_BOT_GEAR_RATIO, 0, 1.39, 10, false);
+  PracticeBase2025(
+      3.65,
+      Elevator.PRACTICE_BOT_GEAR_RATIO,
+      0,
+      1.39,
+      14,
+      false,
+      MotorDirection.CLOCKWISE_POSITIVE),
+  CompetitionBase2025(
+      3.65,
+      Elevator.COMPETITION_BOT_GEAR_RATIO,
+      0,
+      1.39,
+      10,
+      false,
+      MotorDirection.COUNTER_CLOCKWISE_POSITIVE),
+  ;
 
   private final double mass;
   private final double gearRatio;
   private final double minHeight;
   private final double maxHeight;
   private final int motorID;
   private final boolean resetEncoderWhenStowed;
+  private final MotorDirection motorDirection;
 
   ElevatorParameters(
       double mass,
       double gearRatio,
       double minHeight,
       double maxHeight,
       int motorID,
-      boolean resetEncoderWhenStowed) {
+      boolean resetEncoderWhenStowed,
+      MotorDirection motorDirection) {
```

Comment:
The `gearRatio`, `motorID` and `motorDirection` parameters should be grouped together but that can be addressed in a subsequent PR.